spec/acceptance/hieradata/common.yaml

```diff
@@ -1,3 +1,3 @@
 ---
```

Comment:
the rename seems wrong, the package name is not common but specific to the RedHat family

Reply:
I was going to say that https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#hiera shows the default hierachy, but this module still uses the old voxpupuli-acceptance 2.x where `os` really is `family`. See https://github.com/voxpupuli/voxpupuli-acceptance/commit/650b6428205de4a01d6de0fd5e71881b7cfec7a0

Reply:
it does not since yesterday :)